Update syntax for the saveTableToContext step (#3845)
vividus-framework/vividus: docs/modules/commons/pages/variables.adoc

|step                                                                |
|When I go to the relative URL '${optionalPathToCheck:/defaultPath}' |
----

=== Complex types variables

> Vividus provides the ability to work with not only primitive variables like numbers and strings but also supports complex types such as lists and maps.

Reviewer: ```suggestion
VIVIDUS provides the ability to work with not only primitive variables like numbers and strings but also supports complex types such as lists and maps.
```

==== Accessing values

* Values from list variables can be accessed by index. For example, `+${listVar[0]}+` returns the first value.

> * Values from map variables can be accessed by key. For instance, `+${listVar.key1}+` returns the value associated with 'key1'.

Reviewer: ```suggestion
* Values from map variables can be accessed by key. For instance, `+${mapVar.key}+` returns the value associated with 'key'.
```

==== Working with tables

> Vividus provides the ability to store tables in variables.

Reviewer: ```suggestion
VIVIDUS provides the ability to store tables in variables.
```

Tables should have a header and at least one row with values.

.Assuming we have the table below stored in the variable 'tableVar':
[options="header",cols="2,2"]
|===
| A  | B
| A1 | B1
| A2 | B2
|===

> To access specific cell values, specify the row and header key. Rows indexing started from 0.

Reviewer: ```suggestion
To access specific cell values, specify the row and header key. Rows indexing starts from 0.
```

|step                                                                |
|When I go to the relative URL '${optionalPathToCheck:/defaultPath}' |
----

=== Complex types variables

> Vividus provides the ability to work with not only primitive variables like numbers and strings but also supports complex types such as lists and maps.

Reviewer: our users are not technical, they might not know data structures, thus we should avoid describing tables via lists and maps

Author: our users can use conditions and loops, evaluate expressions and JS scripts, but will they be frustrated by lists and maps?

Author: updated
